File: store/src/main/java/in/testpress/store/models/PricesItem.java
package in.testpress.store.models;

import com.google.gson.annotations.SerializedName;

public class PricesItem{

	@SerializedName("end_date")
	private String endDate;

	@SerializedName("price")
	private String price;

	@SerializedName("name")
	private String name;

	@SerializedName("id")
	private int id;

	@SerializedName("validity")
	private Object validity;

	@SerializedName("start_date")
	private String startDate;

	public String getEndDate(){
		return endDate;
	}

	public String getPrice(){
		return price;
	}

	public String getName(){
		return name;
	}

	public int getId(){
		return id;
	}

	public Object getValidity(){
		return validity;
	}

	public String getStartDate(){
		return startDate;
	}
}

File: store/src/main/java/in/testpress/store/models/PricesItem.kt
package `in`.testpress.store.models

data class PricesItem(
    var id: Int? = null,
    var name: String? = null,
    var price: String? = null,
    var validity: Int? = null,
    var endDate: String? = null,
    var startDate: String? = null
)